src/inkpy/backends/libre.py

```diff
@@ -0,0 +1,52 @@
+import six
+import sys
+
+if six.PY2:
```

Comment:
KISS - why supporting 2 python versions if core (only) backend supports only python 3.x ? 

Ralph on the other hand can communicate with the other services no matter which version they use.

So - to simplify things, maybe just focus on one python version?